bin/sozo/src/commands/mod.rs

```diff
@@ -63,6 +65,8 @@ pub enum Commands {
     Model(Box<ModelArgs>),
     #[command(about = "Inspect events emitted by the world")]
     Events(Box<EventsArgs>),
+    #[command(about = "Verify a contract")]
+    Verify(Box<VerifyArgs>),
```

Comment:
This command is too generic, since we don't know if it's explorer verify or walnut.
Could you please introduce again a `walnut` feature and actually gate this argument under the feature with a name like `WalnutVerify` for instance?